### PR TITLE
Setup Migrations

### PR DESCRIPTION
**Pull request by zaher-aa**

### CODE REVIEW

File: src/database/index.ts [hunk 1]
```diff
@@ -8,6 +8,7 @@ import {
   User,
   Watches,
 } from './models';
+// import { migrator, seeder } from './umzug'
```
Comment (AymanSami98) [inline src/database/index.ts:11]: Remove unused comments

**Review by AymanSami98 — APPROVED** [review]

**Review by RandSohail — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -9,7 +9,8 @@
     "build:db": "cross-env NODE_ENV=development ts-node-dev ./src/database/fakeData/index.ts",
     "test": "cross-env NODE_ENV=test jest -i",
     "build": "tsc -p .",
-    "lint": "eslint --ext .ts ./src"
+    "lint": "eslint --ext .ts ./src",
+    "umzug": "cross-env NODE_ENV=development ts-node-dev ./src/database/umzug.ts"
```
Comment (Jood80) [inline package.json:13]: Use a descriptive script, `umzug` means nothing really.
`migration:up`, `migration:generate` maybe, or any other preference of yours.
Also if `umzug` gives you a `umzug-cli` solution, is it possible to use it here in your `scripts` directly to create a separate `migration:up` & `migration:down` scripts?

**Review by Jood80 — COMMENTED** [review]
LGTM

**Review by Ibrahim-Jarada — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (zaher-aa) [reply package.json]: Done

**Pull request merged**